public_html/json_schema_build.php

```diff
@@ -179,7 +179,7 @@ function validate_cache_id($cache_id){
         <li>The <i class="fas fa-book help_text_icon"></i> icon shows whether help text has been written. To add, click on it.</li>
         <li>Be a power user with keyboard shortcuts!
             <ul class="small">
-                <li>Use <code class="border shadow-sm">Enter</code> and <code class="border shadow-sm">Tab</code>+<code class="border shadow-sm">Enter</code> to go up and down.</li>
+                <li>Use <code class="border shadow-sm">Enter</code> and <code class="border shadow-sm">Shift</code>+<code class="border shadow-sm">Enter</code> to go up and down.</li>
```

Comment:
Also, you say `Enter` and `Shift` + `Enter`, to go up and down, but it's actually to go down and up.
 But I agree it sounds better that way.

Reply:
Hah, good point! But yeah, that sounds kind of odd..